guide/sapphire/first-command.md

```diff
@@ -0,0 +1,94 @@
+# Your first command
+
+Now that you have created the `commands` folder, you're ready to make your first command file! In Sapphire, we usually define one command per file, but you can also define groups of them. Hop over to your `commands` folder, and create a new file, we'll show how to make a ping command, so we will name this `ping.js`. This is going to be a simple command that replies to a message and then edits with the elapsed time. We'll go into arguments and other features later.
+
+Once you have your file, it's time to get started!
+
+## Creating your command class
+
+Before you do anything, at the start of your file, you're going to need to require Sapphire Framework again, specifically its Command class.
+
+:::: code-group
+::: code-group-item CommonJS
+```js
+const { Command } = require('@sapphire/framework');
+```
+:::
+::: code-group-item ESM
+```js
+import { Command } from '@sapphire/framework';
+```
+:::
+::::
+
+Commands are classes that are exported in any of the following ways:
+
+:::: code-group
+::: code-group-item CommonJS
+```js
+module.exports = class Foo extends Piece {};
+exports.default = class Foo extends Piece {};
+exports.myCommand = class Foo extends Piece {};
+```
+:::
+::: code-group-item ESM
+```js
+export default class DefaultFoo extends Piece {}
+export class Foo extends Piece {}
+```
+:::
+::::
+
+Because its spread usage, we will be using `module.exports` in all the guides, so let's create the class and set `module.exports` to it!
+
+```js
+module.exports = class PingCommand extends Command {
+	constructor(context) {
+		super(context, {
+			aliases: ['pong'],
+			description: 'Tests the latency.',
+		});
+	}
+};
+```
+
+Don't let this scare you; it's straightforward.
+
+- `context` is an object that contains some file metadata required by the `Command` and `Piece` (which `Command` extends) require in order to function.
+- `name` is, by default, the name of the file without the extension, e.g. `ping.js` becomes `ping`, so we don't need to define it.
+- `aliases` are other ways users can call the command. You can have as many as you want!
+- `description` is the help text displayed when somebody uses a help command.
+
+There are many more properties you can use. The upcoming sections will explain those.
```

Comment:
```suggestion
An overview of what's defined in the constructor:

- `context`: an object that contains file metadata required by the `Command` and `Piece` classes (which `Command` extends) in order to function.
- `name`: by default, the name of the file without the extension, i.e. `ping.js` becomes `ping`, so there's no need to define it.
- `aliases`: other ways users can call the command. You can have as many as you want!
- `description`: the text displayed when the help command is used.

There are many other properties available, explained in upcoming sections.
```

Reply:
Perhaps "required by the `Command` and `Piece` classes (which `Command` extends)" should be a little rewritten here, as the note in parenthesis is meanat to be next to "Piece", unless that's valid.

Reply:
So,
```suggestion
An overview of what's defined in the constructor:
`context`: an object containing file metadata that the `Command` and `Piece` (which is extended by `Command`) classes require to function.
- `name`: by default, the name of the file without the extension, i.e. `ping.js` becomes `ping`, so there's no need to define it.
- `aliases`: other ways users can call the command. You can have as many as you want!
- `description`: the text displayed when the help command is used.
There are many other properties available, explained in upcoming sections.
```
?

Reply:
Maybe "which is extended by the former"?

Reply:
I updated it to "an object that contains file metadata required by the `Piece` class (which `Command` extends)". This seems more straightforward and less wordy